core/controllers/concept_card_viewer_test.py

```diff
@@ -37,10 +37,11 @@ def setUp(self):
 
         self.skill_contents = skill_domain.SkillContents(
             state_domain.SubtitledHtml(
-                '1', '<p>Skill Explanation</p>'), [
-                    state_domain.SubtitledHtml('2', '<p>Example 1</p>'),
-                    state_domain.SubtitledHtml('3', '<p>Example 2</p>')],
-            {'1': {}, '2': {}, '3': {}},
+                '1', 'Skill Explanation'), [
```

Comment:
Should we be changing this? I thought having the `<p>` tags was intended for html fields, but I might be misremembering.

Reply:
Yeah, we are keeping all the HTML inside a tag. (I changed this while resolving merge conflicts.)

Reply:
So it should have p tags, right?